src/app/features/section/shared/list/list-item/member-list-item.component.ts

```diff
@@ -0,0 +1,21 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { User } from '../../interfaces';
+
+@Component({
+  selector: 'app-members-list-item[user]',
+  standalone: true,
+  templateUrl: './member-list-item.component.html',
+  styleUrls: ['./member-list-item.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MemberListItemComponent {
+  @Input() user!: User;
+
+  getFullName(user: User): string {
+    return `${user.firstName} ${user.lastName}`;
+  }
+
+  getInitials(user: User): string {
+    return `${user.firstName.charAt(0)}${user.lastName.charAt(0)}`;
+  }
```

Comment:
te dwie metody powtarzają się z tym co robi @jBachanek-02 dodajmy proszę do backlogu task techniczny na zbudowanie dwóch `pipe` na to